sites.json

```diff
@@ -1,78 +1,88 @@
+      "george-lakoff.com",
+      "glycerinequeenmedia.com",
+      "acearo.at"
+    ],
+    "descriptions": {
+      "aaronparecki.com": "",
+      "alexsirac.com": "A French and English blog that's not about tech.",
+      "boffosocko.com": "A digital commonplace book on education, IndieWeb, mathematics, information theory, biology, history, economics, humanities, note taking & related topics.",
+      "david.shanske.com": "",
+      "jamesg.blog": "Hello! Привіт! I'm James (/d͡ʒeɪmz/) (also capjamesg). My pronouns are he/him/his. I live in Scotland. This is my blog.",
+      "gregorlove.com": "Website of gRegor Morrill, a.k.a. gRegorLove. I’m all about music, faith, computer geekery, friends, and liberty. All while being really, really ridiculously good-looking.",
+      "alabut.com/writing": "",
+      "www.anildash.com": "A blog about making culture. Since 1999.",
+      "reillyspitzfaden.com/blog": "A blog about composition/sound design in Max/MSP, C++ audio plugin development, web development, and other personal interests of mine.",
+      "pcarrier.com": "Personal website and forest of links",
+      "benji.dog": "🇪🇨 benji. not the dog.",
+      "lifeofpablo.com/blog": "Blog",
+      "noisydeadlines.net": "\"I love deadlines. I like the whooshing sound they make as they fly by.\" - Douglas Adams",
+      "tracydurnell.com": "",
+      "tantek.com": "",
+      "artlung.com/blog": "",
+      "hamatti.org/blog": "A blog about technology, communities and life with weekly posts.",
+      "www.lazaruscorporation.co.uk/blogs/artists-notebook": "The Artist’s Notebook",
+      "blog.rtwilson.com": "Robin's Blog, covering geospatial analysis, satellite imaging, Python and more",
+      "omeratagun.net": "Tech oriented blog",
+      "asexualagenda.wordpress.com": "Multi-author blog about asexuality",
+      "going-medieval.com": "Medieval history",
+      "captainawkward.com": "Answering questions about social skills",
+      "sarahspiekermann.com": "From professor one of whose topics is ethical technology",
+      "ialja.com": "About responsible tech",
+      "devastatia.com": "Entertaning blog just for the tone",
+      "matuzo.at": "HTML, CSS and accesibility",
+      "nineteenteen.blogspot.com": "19th century history",
+      "sloanime.si": "Blog about Slovenian anime scene",
+      "katetattersall.com": "Victorian secret service blog",
+      "thisvictorianlife.com": "Blog about Victorian era",
+      "criticalpolyamorist.com": "About indigenous, racial, and cultural politics related to open non-monogamy",
+      "mythcreants.com": "Fiction writing",
+      "george-lakoff.com": "Mostly political blog from professor known for his work on methaphorical language",
+      "glycerinequeenmedia.com": "Queer and other stuff",
+      "acearo.at": "Austrian asexual and aromantic oriented blog"
+    },
+    "code": "en"
+  },
+  "zh": {
+    "blogs": ["yinji.org"],
+    "descriptions": {
+      "yinji.org": ""
+    },
+    "code": "zh"
```

Comment:
This structure will work, but I definitely feel like it's possible with string keys we could get out of sync in each sub-area. I think we need a script that doublechecks that if we go with the heavier structure. I kind of think it would be simpler to do `sites.en.json` and `sites.zh.json` with a "blogs" and "descriptions" top level keys